src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -28,27 +28,26 @@
                         data-transition-leave-to="opacity-0 scale-95"
                         tabindex="-1"
                     >
-                        <div class="space-y-2" {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
+                        <div {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
                             {% if table.exporters|length > 1 %}
-                                <p class="whatwedo-utility-heading-3 mb-3">{{ 'araise_table.download.choices' | trans }}</p>
-                                <div>
-                                    <ul role="list">
-                                        {% for acronym,exporter in table.exporters %}
-                                            <li>
-                                                <input
-                                                    type="checkbox"
-                                                    id="{{ acronym }}"
-                                                    name="exporter"
-                                                    value="{{ acronym }}"
-                                                    {% if table.exporters|length > 1 %} {{ stimulus_action('araise/table-bundle/exporter', 'select')}} {% endif %}
-                                                >
-                                                <label for="{{ acronym }}">{{ exporter.label | trans }}</label>
-                                            </li>
-                                        {% endfor %}
-                                    </ul>
-                                </div>
+                                <p class="whatwedo-utility-heading-3 mb-2">{{ 'araise_table.download.choices' | trans }}</p>
+                                <ul class="mb-4" role="list">
+                                    {% for acronym,exporter in table.exporters %}
+                                        <li class="flex items-center mb-2 gap-x-2">
+                                            <input
+                                                class="whatwedo_core-checkbox"
+                                                type="checkbox"
+                                                id="{{ acronym }}"
+                                                name="exporter"
+                                                value="{{ acronym }}"
+                                                {% if table.exporters|length > 1 %} {{ stimulus_action('araise/table-bundle/exporter', 'select')}} {% endif %}
```

Comment:
Same with the inline if-else. We need to find another way.

Reply:
I just did a little improvement by moving the if/else statement out of the html element.

In the future a helper could be created like this:

```
{% set attributes = {
    'id': acronym,
    'name': 'exporter',
    'value': acronym,
} %}
<input{{ html_attributes(attributes) }}>
```

Out of scope now. But noted